index.js

```diff
@@ -1,48 +1,30 @@
 const path = require('path');
 const decache = require('decache');
+const squba = require('squba')
 
-const requireReg = /require\s*\(['|"](.+)['|"]\)(?:\.([^;\s]+))?[;\s]/g;
+const requireReg = /require\s*\((["'])([\w.\/]+)(?:\1)\)((?:\.[\w_-]+)*);?/igm;
```

Comment:
It is really cool that we can get nested properties. The thing is that I tried it in our project, and it failed, because the property got by the new regex started with a dot (like '.default'). I tried it with this modified regex and worked: ```/require\s*\((["'])([\w.\/]+)(?:\1)\)\.((?:[\w_-]+)*);?/igm``` (I moved the ```\.``` out of the group)

Reply:
Strange, it is working for me I think (although I will double check now). At a glance that modified regex looks like it will not match a `require("...");` with no dot-part. I'll sort it out one way or another, anyway, I agree it is a bit nasty passing the key string all the way through to `squba` with the `.` still stuck on the front.

Reply:
:+1: